src/header.c

```diff
@@ -122,14 +122,11 @@ static void set_matches_and_header(enum header_type hdr_type,
 	char **header, enum yall_matches **matches)
 {
 	switch (hdr_type) {
-	case std_header:
-		*matches = std_matches;
-		*header = std_header_format;
-		break;
 	case call_header:
 		*matches = call_matches;
 		*header = call_header_format;
 		break;
+	case std_header:
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant "case" clause. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=c%3AS3458)